### PR TITLE
Fix dev requirements install

### PR DESCRIPTION
**Pull request by un4gt**
The command `pip install -e '.[dev]'` results in a syntax error on some systems due to single quotes. 

In windows :

```
pip install -e '.[dev]'
ERROR: '.[dev]' is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with bzr+http, bzr+https, bzr+ssh, bzr+sftp, bzr+ftp, bzr+lp, bzr+file, git+http, git+https, git+ssh, git+git, git+file, hg+file, hg+http, hg+https, hg+ssh, hg+static-http, svn+ssh, svn+http, svn+https, svn+svn, svn+file).
```

By using double quotes (`pip install -e ".[dev]"`), the command runs successfully on all platforms, ensuring better cross-platform compatibility.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -103,3 +103,10 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# PyCharm
+#  JetBrains specific template is maintained in a separate JetBrains.gitignore that can
+#  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
+#  and can be added to the global gitignore or merged into this file.  For a more nuclear
+#  option (not recommended) you can uncomment the following to ignore the entire idea folder.
+.idea/
```
Comment (lafrech) [inline .gitignore]: This doesn't belong in the project but in [your own .gitignore](https://stackoverflow.com/a/7335487/) (local to you computer, global to your projects).

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -48,7 +48,7 @@ Setting Up for Local Development
 ::
 
     # After activating your virtualenv
-    $ pip install -e '.[dev]'
+    $ pip install -e ".[dev]"
```
Comment (lafrech) [inline CONTRIBUTING.rst:51]: It even works without any quotes on my machine (Linux). Would that work on Windows?

    pip install -e .[dev]

**Review by lafrech — CHANGES_REQUESTED** [review]

**Comment by lafrech** [comment]
The CI failure is unrelated and seems to be due to changes in bottle 0.13.

**Comment by sirosen** [comment]
> The CI failure is unrelated and seems to be due to changes in bottle 0.13.

I'll plan to look into that either today or tomorrow.

**Comment by lafrech** [comment]
Thanks Stephen.

It seems related to how unicode errors are treated but I haven't gotten to the bottom of it and I gotta go.

I scrolled through [bottle changelog](https://bottlepy.org/docs/dev/changelog.html#release-0-13) and nothing stroke me as relevant to this issue.

File: CONTRIBUTING.rst (hunk 2 above)
Comment (un4gt) [reply CONTRIBUTING.rst:51]: single quote  is notworking on windows:

image

But, double quote and without quote works well.

I think we need to unify the commands.

Comment (lafrech) [reply CONTRIBUTING.rst:51]: Sure. I don't know why we put those quotes in the first place so I'd go for no quotes at all if it works everywhere.

Comment (sloria) [reply CONTRIBUTING.rst:51]: oops, on zsh the quotes are required

```
$ pip install -e .[dev]                              
zsh: no matches found: .[dev]
```

double quotes seems to work though, so i'm fine with this change as is

**Review by sloria — APPROVED** [review]

File: CONTRIBUTING.rst (hunk 2 above)
Comment (lafrech) [reply CONTRIBUTING.rst:51]: OK so that's why we put those quotes in the first place. Double quotes is fine, then.

**Pull request merged**